Dockerfile

```diff
@@ -19,6 +19,8 @@ FROM ubuntu:18.10
 RUN apt-get update
 RUN apt-get install -y openssl
 RUN apt-get install -y net-tools
+RUN apt-get install -y locales
+RUN locale-gen en_US.UTF-8
```

Comment:
You'll also need

```
# We unfortunately cannot use update-locale because docker will not use the env variables
# configured in /etc/default/locale so we need to set it manually.
ENV LANG=en_US.UTF-8
```